index.html

```diff
@@ -1571,7 +1583,14 @@ <h3>Proofs (Signatures)</h3>
 
         <p class="note">
 No single proof mechanism is currently standardized or recommended for
-<a>verifiable credentials</a>.
+<a>verifiable credentials</a>. Implementers that are interested in
+understanding more about the <code>proof</code> mechanism can
+learn more by reading the following specifications:
+Linked Data Proofs [[LD-PROOFS]], Linked Data Signatures [[LD-SIGNATURES]],
+2018 RSA Signature Suite [[LDS-RSA2018]], and JSON Web Signature (JWS)
+Unencoded Payload Option [[RFC7797]]. A current list of available proof
+mechanisms is available in the Verifiable Credentials Extension Registry
```

Comment:
same as above, recommend changing to:
"A list of proof mechanisms is available in the Verifiable Credentials . . ."

Reply:
Fixed in 4816ada7ed318fa2f13888569581376a0f71804d.